src/System.Diagnostics.Process/tests/ProcessStartInfoTests.cs

```diff
@@ -512,6 +503,190 @@ public void TestVerbsProperty()
             Assert.DoesNotContain("closed", psi.Verbs, StringComparer.OrdinalIgnoreCase);
         }
 
+        [Theory]
+        [InlineData("")]
+        [InlineData("nofileextension")]
+        [PlatformSpecific(TestPlatforms.Windows)]
+        public void Verbs_GetWithNoExtension_ReturnsEmpty(string fileName)
+        {
+            var info = new ProcessStartInfo { FileName = fileName };
+            Assert.Empty(info.Verbs);
+        }
+
+        [Fact]
+        [PlatformSpecific(TestPlatforms.Windows)]
+        public void Verbs_GetWithNoRegisteredExtension_ReturnsEmpty()
+        {
+            var info = new ProcessStartInfo { FileName = "file.nosuchextension" };
+            Assert.Empty(info.Verbs);
+        }
+
+        [Fact]
+        [PlatformSpecific(TestPlatforms.Windows)]
+        public void Verbs_GetWithNoEmptyStringKey_ReturnsEmpty()
+        {
+            const string Extension = ".noemptykeyextension";
+            const string FileName = "file" + Extension;
+
+            try
+            {
+                Registry.ClassesRoot.CreateSubKey(Extension);
```

Comment:
Not sure whether any (?) of our outer loop runs are elevated/sudo. I think someone mentioned the Helix runs are, but it was a bug. @MattGal can you remind us?